### PR TITLE
add message to email support for help if account is cancelled

### PR DESCRIPTION
**Pull request by zacjones93**
cry for help

show cancel message

### CODE REVIEW

File: src/components/pages/user/components/subscription-details.tsx [hunk 1]
```diff
@@ -61,8 +61,18 @@ const SubscriptionDetails: React.FunctionComponent<SubscriptionDetailsProps> =
                   </strong>
                   ). You can renew at any time.
                 </p>
-                <p></p>
-                <p></p>
+                <p>
+                  You can also get help by emailing{' '}
+                  <strong>
+                    <a
+                      href="mailto:support@egghead.io"
```
Comment (joelhooks) [inline src/components/pages/user/components/subscription-details.tsx:68]: mailto links can have a subsject AND body added which should almost always be the case for this type of usage https://mailtolinkgenerator.com/

Comment (DrShpongle) [reply src/components/pages/user/components/subscription-details.tsx:68]: something like:
```
<a
  href={`mailto:support@egghead.io?subject=${encodeURIComponent(
    `Subscription ${subscriptionData.subscription.id} is cancelled`,
  )}&body=${encodeURIComponent(
    `Subscription ${subscriptionData.subscription.id} is cancelled`,
  )}`}
  style={{textDecoration: 'underline'}}
>
  support@egghead.io
</a>
```

File: src/components/users/subscription-details.tsx [hunk 2]
```diff
@@ -97,10 +97,7 @@ const SubscriptionDetails: React.FunctionComponent<SubscriptionDetailsProps> =
                         <p>
                           You still have access to a Pro Membership. If you feel
                           this is in error please email{' '}
-                          <a
-                            className="text-blue-600 underline hover:text-blue-700"
-                            href="mailto:support@egghead.io"
-                          >
+                          <a href="mailto:support@egghead.io?subject=support%20needed%20for%20egghead%20membership">
```
Comment (DrShpongle) [inline src/components/users/subscription-details.tsx:100]: it seems that this component is not used anywhere

File: src/pages/user/membership.tsx [hunk 3]
```diff
@@ -68,8 +68,12 @@ const Account = () => {
           </h2>
           <p className="w-fit mx-auto mb-12">
             If this is incorrect, please reach out to{' '}
-            <strong>support@egghead.io</strong> or your team owner{' '}
-            {accountOwner.email}.
+            <strong>
+              <a href="mailto:support@egghead.io?subject=Support needed for egghead team membership">
```
Comment (DrShpongle) [inline src/pages/user/membership.tsx:72]: I think we want to use `encodeURIComponent` for the strings here

File: src/components/pages/user/components/subscription-details.tsx [hunk 4]
```diff
@@ -61,8 +61,18 @@ const SubscriptionDetails: React.FunctionComponent<SubscriptionDetailsProps> =
...
+                      style={{textDecoration: 'underline'}}
```
Comment (DrShpongle) [inline src/components/pages/user/components/subscription-details.tsx:69]: can be just `underline` class

File: src/pages/user/membership.tsx (hunk 3 above)
Comment (DrShpongle) [reply src/pages/user/membership.tsx:72]: probably not 🤔

File: src/components/users/subscription-details.tsx (hunk 2 above)
Comment (zacjones93) [reply src/components/users/subscription-details.tsx]: ah, wasn't sure

**Review by DrShpongle — APPROVED** [review]
👍

**Pull request merged**